maqs-webservices/src/test/java/com/cognizantsoftvision/maqs/webservices/WebServiceDriverRequestUnitTest/WebServiceDriverPutUnitTest.java

```diff
@@ -4,11 +4,7 @@
 
 package com.cognizantsoftvision.maqs.webservices.WebServiceDriverRequestUnitTest;
 
-import com.cognizantsoftvision.maqs.webservices.HttpClientFactory;
-import com.cognizantsoftvision.maqs.webservices.MediaType;
-import com.cognizantsoftvision.maqs.webservices.WebServiceConfig;
-import com.cognizantsoftvision.maqs.webservices.WebServiceDriver;
-import com.cognizantsoftvision.maqs.webservices.WebServiceUtilities;
+import com.cognizantsoftvision.maqs.webservices.*;
```

Comment:
Why are we moving to the wild card pattern? - Not sure this is a good idea

Reply:
brought back all imports